clients/web/src/views/App.js

```diff
@@ -221,7 +223,8 @@ var App = View.extend({
 
             settings = _.extend(settings, {
                 el: this.$('#g-app-body-container'),
-                parentView: this
+                parentView: this,
+                brandName: this.brandName
```

Comment:
Yes, a settings registry is way out of scope here, but if we agree it's a good idea, we should get it in before the next release, as the current implementation of passing things to every body view isn't very clean.